packages/common-all/src/dnode.ts

```diff
@@ -887,20 +887,16 @@ export class NoteUtils {
     // Separate custom and builtin props
     const builtinProps = _.pick(propsWithTrait, [
       ...Object.values(DNodeExplicitPropsEnum),
-      "stub",
     ]);
 
     const { custom: customProps } = cleanProps;
     const meta = { ...builtinProps, ...customProps };
```

Comment:
Do I understand this correct that a `stub` field can still be part of this object because it could still "live" inside `customProps` but not `buildinProps` since that has now been "unpicked".

Reply:
correct